src/main/java/uk/gov/justice/digital/delius/service/CustodyService.java

```diff
@@ -460,4 +485,11 @@ static PrisonLocationUpdateSuccess noUpdateRequired(final List<Custody> custodyR
         all.putAll(map2);
         return Map.copyOf(all);
     }
+
+    IntegrationContext getContext(String contextName) {
+        var context = deliusIntegrationContextConfig.getIntegrationContexts().get(contextName);
+        return ofNullable(context).orElseThrow(
+            () -> new BadRequestException("IntegrationContext does not exist for: " + contextName)
```

Comment:
This validation is not tested (?)

Reply:
Also should this be an `IllegalStateException`, or something similar?